xla/service/gpu/gpu_compiler.cc

```diff
@@ -114,6 +114,7 @@ limitations under the License.
 #include "xla/service/gpu/autotuning/custom_kernel_fusion_autotuner.h"
 #include "xla/service/gpu/compile_module_to_llvm_ir.h"
 #include "xla/service/gpu/conv_layout_normalization.h"
+#include "xla/service/gpu/cudnn_custom_call_converter.h"
```

Comment:
There was a BUILD dependency missing for this header included. I am fixing this now, and will also move the pass to the transforms subdirectory. We are in the process of moving all HLO passes from the gpu directory to the transforms subdirectory.